src/PowerSystem/power_network.jl

```diff
@@ -50,17 +50,45 @@ struct PowerNetwork <: AbstractPowerSystem
         gen = data["gen"]
         lines = data["branch"]
         SBASE = data["baseMVA"][1]
-        costs = Base.get(data, "cost", nothing)
+        cost_coefficients = Base.get(data, "cost", nothing)
 
-        bus_id_to_indexes = get_bus_id_to_indexes(data["bus"])
+        # BUSES
+        bus_id_to_indexes = get_bus_id_to_indexes(bus)
 
+        # GENERATORS
+        if has_multiple_generators(gen) && multi_generators == :aggregate
+            gen, σg = merge_multi_generators(gen)
+            if !isnothing(cost_coefficients)
+                cost_coefficients = merge_cost_coefficients(cost_coefficients, gen, σg)
+            end
+        end
+
+        # LINES
         # Remove specified lines
         lines = get_active_branches(lines, remove_lines)
 
         # size of the system
         nbus = size(bus, 1)
         ngen = size(gen, 1)
 
+        # COSTS
+        if isnothing(cost_coefficients)
```

Comment:
@frapac In the message. Could you specify the generator number which does not have the coefficients? Also, can we show warnings instead of printing it directly to STDOUT?

Reply:
Below I see that you use the macro "@warn"

Reply:
Again, good catch! I will ensure that the warning are printed in STDERR. 
Just: if `cost_coefficient` is equal to `nothing`, that means that all costs are equal to 0. It happens when we are importing PSSE file, as we are not importing the attached file corresponding to the cost.